docs/dev/table/connectors/formats/json.md

```diff
@@ -136,6 +136,13 @@ Format Options
       <td>String</td>
       <td>Specify string literal to replace null key when <code>'json.map-null-key.mode'</code> is LITERAL.</td>
     </tr>     
+    <tr>
+      <td><h5>json.encode.decimal-as-plain-number</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Encode all decimals as plain numbers instead of possible scientific notations. e.g. <code>0.000000027</code> is encoded as <code>2.7E-8</code> when this option is set to false.</td>
```

Comment:
```suggestion
      <td>Encode all decimals as plain numbers instead of possible scientific notations. By default decimals are written using scientific notation. For example, <code>0.000000027</code> is encoded as <code>2.7E-8</code> by default, and will be written as <code>0.000000027</code> if set this option to true.</td>
```

Reply:
Please also update this?